spec/support/travel_advice_publisher_navigation_helpers.rb

```diff
@@ -0,0 +1,89 @@
+module TravelAdvicePublisherNavigationHelpers
+  def visit_travel_advice_publisher_homepage
+    travel_advice_publisher_url("/admin")
+  end
+
+  def visit_draft_origin_url(path)
+    visit(Plek.find("draft-origin") + path)
+  end
+
+  def visit_published_url(path)
```

Comment:
It seems like most, or every, time this is called we pass `/foreign-travel-advice/`, is it possible to extract that out to a constant or another method.